app/code/Transoft/Blog/view/frontend/layout/blog_blog_index.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<page layout="1column" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="page.main.title">
+            <action method="setPageTitle">
+                <argument name="pageTitle" translate="true" xsi:type="string">Blog</argument>
+            </action>
+        </referenceBlock>
+        <referenceContainer name="content">
+            <block name="blog.singleblog" template="Transoft_Blog::singleBlogTmpl.phtml">
+                <arguments>
+                    <argument name="singleblog_view_model" xsi:type="object">Transoft\Blog\ViewModel\Singleblog</argument>
```

Comment:
Why do we need ViewModel in our own template?

Reply:
this is just to test block appearance at storefront